manifests/s/SnailDOS/StreamBop/1.5.4/SnailDOS.StreamBop.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SnailDOS.StreamBop
+PackageVersion: 1.5.4
+PackageLocale: en-US
+Publisher: SnailDOS
+PublisherUrl: https://github.com/snaildos/StreamBop/tree/main
+PublisherSupportUrl: https://github.com/snaildos/StreamBop/issues
+# PrivacyUrl: 
+Author: SnailDOS
+PackageName: streambop
```

```suggestion
PackageName: StreamBop
```
It's best if capitalization matches